scripts/build.js

```diff
@@ -121,6 +114,19 @@ function writePackage(versionPath, version) {
     )
 }
 
+function writeIndex(versionPath) {
+    fs.writeFileSync(
+        path.resolve(versionPath, "lib", "index.js"),
```

Comment:
This is not critical, but IMHO it's better to write this file somewhere in the codebase and just copy it as part of the build process. It will make sure that we're not making any mistake and makes the code a bit clearer.

Reply:
Yea, was thinking about it but did want to come up with some artificial filename or folder where to keep it. It's such a small thing it's ok.